test/module/irohad/common/CMakeLists.txt

```diff
@@ -6,4 +6,6 @@ target_link_libraries(blob_converter_test
 AddTest(raw_block_loader_test raw_block_loader_test.cpp)
 target_link_libraries(raw_block_loader_test
     raw_block_loader
+    shared_model_proto_backend
```

Comment:
Move these two lines to `irohad/network/CMakeLists.txt` for `target_link_libraries(block_loader`, because they are used there.